WrightTools/artists/_helpers.py

```diff
@@ -487,7 +487,7 @@ def get_scaled_bounds(ax, position, *, distance=0.1, factor=200):
     return [h_scaled, v_scaled], [va, ha]
 
 
-def pcolor_helper(xi, yi, zi):
+def pcolor_helper(xi, yi):
```

Comment:
for backwards compatibility, we may wish to include `zi=None`, with an entry in the docstring stating it does nothing.
We could also invoke a DepricationWarning if it is not `None`

Reply:
pcolor_helper _is_ part of our public API, removing the third positional argument is a breaking change to old code.